src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -123,7 +124,7 @@
      *
      * @var array
      */
-    protected static $dynamicRelations = [];
+    public static $dynamicRelations = [];
```

Comment:
Why is this public?

Reply:
Sorry, was trying to make a feature, and kept it public for debugging-related reasons. Will push a fix right away.

Reply:
Fixed!